### DISCUSSION TITLE
2021/11/09/%E5%9C%A8%E8%80%83%E8%99%91%E8%87%AA%E5%B7%B1%E5%BB%BA%E4%B8%80%E4%B8%AA%E5%8D%9A%E5%AE%A2%E7%BD%91%E7%AB%99

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#359] [Announcements]
# 2021/11/09/%E5%9C%A8%E8%80%83%E8%99%91%E8%87%AA%E5%B7%B1%E5%BB%BA%E4%B8%80%E4%B8%AA%E5%8D%9A%E5%AE%A2%E7%BD%91%E7%AB%99

Wordpress 免费版的功能好多限制，还会插广告，中国也没法访问。如果在自己的服务器上建一个博客，这些问题就都能解决了。

https://blog.qizhen.xyz/2021/11/09/%E5%9C%A8%E8%80%83%E8%99%91%E8%87%AA%E5%B7%B1%E5%BB%BA%E4%B8%80%E4%B8%AA%E5%8D%9A%E5%AE%A2%E7%BD%91%E7%AB%99

**Comment by ruanqizhen** [comment]
之前博客下的留言：

lhb5883
2021年11月9日 22:19 编辑
阮总好啊，可以考虑在github page上搞个hexo静态blog，然后用cloudflare做一个cdn，国内访问速度还凑活。

回复
ruanqizhen
2021年11月10日 08:40 编辑
多谢建议啊，我去研究一下。你的blog就是这么建的吗？

回复
lhb5883
2021年11月15日 03:37 编辑
是的，算是一个比较简单的的方案了，如果还是想用wordpress，也可以用AWS的lightsail直接一键安装Ubuntu+wordpress.
评论系统没有什么好的解决方案，第三方服务都要翻墙我现在是用的github issue，需要注册github账号。

回复
ruanqizhen
2021年11月15日 09:35 编辑
太好了，我最近几天就学习一下github page相关的技术。多谢建议！

回复
ruanqizhen
2021年11月16日 09:35 编辑
你的博客留言系统好像还有点问题，我点击留言，留言区是灰色的，不过上面有个登陆github链接（我其实已经登陆了的），点击这个链接就直接跳转回了你的blog主页面，还是不能留言。

回复
